CONTRIBUTING.md

```diff
@@ -83,6 +83,20 @@ feature. We use the 'fork and pull' model described there.
 
 Please make pull requests against the `master` branch.
 
+Compiling all of `make check` can take a while. When testing your pull request,
+consider using one of the more specialized `make` targets to cut down on the
+amount of time you have to wait.
+
+    $ make -j8 rustc-stage1 && make check-stage1
+
+Is one such example, which builds just `rustc`, and then runs the tests. If
+you’re adding something to the standard library, try
+
+    $ make -j8 check-stage1-std NO_REBUILD=1
+
+This will not rebuild the compiler, but will run the tests. You will still have
```

Comment:
Maybe the last sentence should say something like "You need to have built the compiler at least once before running this", to make it clear that the building is a required active action.